packages/grafana-data/src/field/fieldOverrides.ts

```diff
@@ -50,15 +42,13 @@ export function findNumericFieldMinMax(data: DataFrame[]): GlobalMinMax {
 /**
  * Return a copy of the DataFrame with all rules applied
  */
-export function applyFieldOverrides(
-  data: DataFrame[],
-  source: FieldConfigSource,
-  replaceVariables: InterpolateFunction,
-  theme: GrafanaTheme,
-  isUtc?: boolean
-): DataFrame[] {
+export function applyFieldOverrides(options: GetFieldDisplayValuesOptions): DataFrame[] {
```

Comment:
I am a bit torn changing this signature because the "source" value becomes `FieldDisplayOptions` rather than the more limited version...

But passing the other values is pretty messy when the rest is all the same.

Lets keep an eye on this and aim to clean it up as we apply more patterns to it